prog/vnet/update_load_balancer_node.rb

```diff
@@ -29,6 +41,68 @@ def before_run
     pop "load balancer is updated"
   end
 
+  label def put_certificate
+    cert = load_balancer.active_cert
+    cert_payload = cert.cert
+    cert_key_payload = OpenSSL::PKey::RSA.new(cert.csr_key).to_pem
+    vm.vm_host.sshable.cmd("sudo -u #{vm.inhost_name} mkdir -p #{cert_folder}")
+    vm.vm_host.sshable.cmd("sudo -u #{vm.inhost_name} tee #{cert_path}", stdin: cert_payload)
+    vm.vm_host.sshable.cmd("sudo -u #{vm.inhost_name} tee #{key_path}", stdin: cert_key_payload)
+    hop_start_certificate_server
+  end
+
+  label def start_certificate_server
+    service_name = "#{vm.inhost_name}_cert_server"
```

Comment:
We use `-` in dnsmasq service. 
```suggestion
    service_name = "#{vm.inhost_name}-cert-server"
```